composer.json

```
@@ -1,6 +1,6 @@
 {
-  "name": "wordpress/ai",
-  "description": "Experimental AI features for WordPress",
+  "name": "wordpress/ai-experiments",
```

Given our other conversation, I think this slug should just be `ai` as well.
```suggestion
  "name": "wordpress/ai",
```